chromium/browser_type.go

```diff
@@ -195,7 +195,12 @@ func (b *BrowserType) launch(
 		return nil, 0, fmt.Errorf("launching browser: %w", err)
 	}
 
-	return browser, browserProc.Pid(), nil
+	pid, err = browserProc.Pid()
+	if err != nil {
+		return nil, 0, fmt.Errorf("launching browser: %w", err)
+	}
```

Comment:
Wouldn't this always return a valid PID?

Reply:
Knowing the current implementation behind `allocate` and `NewLocalBrowserProcess` yes, but handling the error seems the proper way if we maintain this API for `Pid`.
Would you ignore the error handling based on the knowledge of the mentioned methods?

Reply:
I would eliminate the interface and never need to check for errors.